src/esp32/board.rs

```diff
@@ -1,53 +1,228 @@
 #![allow(dead_code)]
-use super::pin::PinExt;
```

Comment:
Does `PinExt` still serve a purpose?

Reply:
Yes, we need this to construct motor toe for example. In practice we could do away with it but that would mean making the ESP32Motor only work on esp32 so we would have to re-implement it for each new platform we support. I would ideally keep it abstracted if we can it just unclear for now if we can actually achieve this